src/Prima/Pyxis/Form/Autocomplete.elm

```diff
@@ -618,6 +743,16 @@ renderAutocompleteNoResults =
     ]
 
 
+{-| Internal. Renders the `AutocompleteChoice` list loading.
+-}
+renderAutocompleteLoading : List (Html Msg)
```

Comment:
Qui è meglio chiedere ai designers se non sia il caso di inserire un'icona `spinner` dentro il componente. Non mi piace molto l'idea di avere la lista con scritto "Aggiornamento". Puoi chiedere a @francescophra un consulto in merito.

Reply:
@Gsciarra ho dato un'occhiata al componente, secondo me la scelta che hai fatto sul loading funziona già così ma sono d'accordo con @alessiosantangelo che una lista con solo `Aggiornamento` si possa migliorare.

Potremmo ad esempio mostrare uno spinner al posto dell'icona di search quando viene triggerato il loading. 
In questo modo la lista delle options dell'autocomplete resta nascosta fin quando non vengono restituiti i risultati. Una volta che abbiamo le options a disposizione possiamo nascondere lo spinner e mostrare nuovamente l'icona di search.

Reply:
@francescophra ho integrato la logica che mi hai suggerito, ma non sono riuscito a capire come inserire uno spinner al posto dell'icona del search. Per il momento ho inserito la classe .form-autocomplete__spinner-icon (non funzionante) che si alterna con la preesistente .form-autocomplete__search-icon (funzionante).

Reply:
@Gsciarra su Pyxis NPM abbiamo un tot di mixins sotto `icons.scss` che renderizzano delle icone pure-css. Tra queste c'è anche `iconLoader` che puoi includerti nel tuo selettore.